app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1425,6 +1449,15 @@ DeviceDelegateOpenXR::UpdateHandMesh(const uint32_t aControllerIndex, const std:
                const vrb::GroupPtr& aRoot, const bool aEnabled, const bool leftHanded) {
   if (!m.handMeshRenderer)
     return;
+
+  if (m.handMeshMSFT.isSupported) {
+    HandMeshBufferPtr buffer = m.input->GetHandMeshBuffer(aControllerIndex);
+    if (buffer) {
+      m.handMeshRenderer->Update(aControllerIndex, handJointTransforms, aRoot, buffer, aEnabled, leftHanded);
+      return;
+    }
+  }
```

Comment:
Ok, I left only the version of `Update()` with the buffer. Now we fetch a buffer first, unconditionally, and pass it to Update(). The strategies that don't need it will just assert that it is null.